### PR TITLE
[next] Try named imports

### PR DESCRIPTION
**Pull request by allanbreyes**
#4827 

~**NOTE**: pushing this up early, will rebase/squash the most recent commit. Can't seem to get tests to run locally, at the moment. (Likely PEBKAC.)~

For the curious, here's `core`'s dependency graph:

image

(No change before and after, which is a good sign for parity.)

### CODE REVIEW

**Comment by bigtimebuddy** [comment]
You need to also remove `export default class` and change to `export class`

File: README.md [hunk 1]
```diff
@@ -175,6 +175,13 @@ npm run docs
 
 The documentation uses [Jaguar.js](https://github.com/pixijs/jaguarjs-jsdoc) and the jsdoc format, the configuration file can be found at [scripts/jsdoc.conf.json](scripts/jsdoc.conf.json)
 
+### How to set up for development ###
+
+As of v5, PixiJS uses [Lerna](https://github.com/lerna/lerna) for development.
+To get started, install Lerna globally (`npm install -g lerna`), and run
+`lerna bootstrap`. This will install all the dependencies for all packages in
+this repository.
```
Comment (bigtimebuddy) [inline README.md]: I would like to remove this note about installing Lerna globally. Lerna is a devDependency and should work with `npm install` (this runs bootstrapping). Other than that we don't use Lerna that much.

**Comment by allanbreyes** [comment]
Ah good call, just found that out after running `npm test`. Currently trying to figure out what's up with Rollup and this namespace collision:

```
bundles/pixi.js/src/index.js → bundles/pixi.js/dist/pixi.js...
[!] Error: 'Resource' is not exported by packages/loaders/node_modules/resource-loader/lib/index.js
https://github.com/rollup/rollup/wiki/Troubleshooting#name-is-not-exported-by-module
packages/loaders/lib/loaders.es.js (8:25)
 6:  * http://www.opensource.org/licenses/mit-license
 7:  */
 8: import ResourceLoader, { Resource } from 'resource-loader';
                             ^
 9: import EventEmitter from 'eventemitter3';
10: import { Texture } from '@pixi/core';

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @ build: `rollup -c`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @ build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/allan/.npm/_logs/2018-04-13T02_58_39_405Z-debug.log
npm ERR! Test failed.  See above for more details.
```

File: README.md (hunk 1 above)
Comment (allanbreyes) [reply README.md]: :+1: will remove and squash

**Comment by allanbreyes** [comment]
Hmm... despite the passing test suite, I'm not confident that this isn't a failing build. Locally, I'm still getting the same error, despite patching `packages/loaders/src/TextureLoader.js` and `packages/loaders/src/index.js` for the named `{ Resource }` import. Not quite sure why...

**Pull request closed without merging**